openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -867,17 +879,33 @@ def test_user_with_permission_deactivates_user(self):
             )
         )
         user.user_permissions.add(permission)  # pylint: disable=no-member
+        headers = self.build_jwt_headers(user)
         self.client.login(username=user.username, password=TEST_PASSWORD)
         self.assertTrue(self.test_user.has_usable_password())  # pylint: disable=no-member
```

Comment:
It is, this line just tests if the user has a usable password before deactivating it's account. I'll move the line to DRY it a little bit.